packages/primitives/src/Select/Select.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react'
+import styled from '@emotion/styled'
+import { Theme, css } from 'theme-ui'
+import { SerializedStyles } from '@emotion/serialize'
+import {
+  StyledSystemProps,
+  variant,
+  sx,
+  shouldForwardProp,
+  styledSystemProps,
+  Box,
+} from '../Box/Box'
+import { Flex } from '../Flex'
+
+/**
+ * base
+ *
+ * Generate the base CSS for the button component
```

Comment:
I think I've copy pasted these comments as well so I'll have to check for them in the other components but I just noticed they refer to button 👆 and Box 👇

Reply:
ah yea good catch!